dev/WindowsAppSDK_Insights/WindowsAppSDKInsightsLib/WindowsAppSDKInsightsLib.vcxproj

```diff
@@ -134,6 +135,7 @@
       <SDLCheck>true</SDLCheck>
       <PreprocessorDefinitions>WIN32;NDEBUG;_LIB;%(PreprocessorDefinitions)</PreprocessorDefinitions>
       <ConformanceMode>true</ConformanceMode>
+      <RuntimeLibrary>MultiThreadedDebug</RuntimeLibrary>
```

Comment:
>MultiThreadedDebug [](http://example.com/codeflow?start=22&length=18)

Release should not reference MTD library.

Reply:
Done. Will be in next iteration.